cip-0028.md

```diff
@@ -0,0 +1,41 @@
+    CIP: 28
+    Title: Updated AT max steps
+    Author: frank_the_tank, jjos
+    Comments-URI: https://github.com/burst-apps-team/burstcoin/commit/123e26a3ded62f184f70f2d40e18ea5fd8b938f1#diff-de81aa4fb23d09bcc765945ed2d84d7bc10d76925338e98c2a48c77da8c8cc4d
+    Status: Draft
+    Type: Enhancement (Hard Fork)
+    Created: 2021-03-25
+
+## Motivation
+
+Burst smart contracts became much more useful with [CIP-20](cip-0020.md). However, it became evident that, for more complex applications, the maximum number of steps a contract is allowed to run would need to increase.
+This is a simple proposal increasing the maximum number of steps a contract can run in a single block to `1'000'000`.
+
+This is a tenfold increase compared to the current value and the impact on nodes should be minimum for any modern computer.
+For many cases this will even be benefical for the network, since allowing contracts to finish their running tasks in the same block reduces the amount of information that needs to be stored permanently on the blockchain.
+It should also be stressed that a fee is paid for every single step a contract runs.
+
+## Specification
+
+In the next hard-fork event, use the following code:
+```java
+    public long maxSteps(int height) {
+        if(Burst.getFluxCapacitor().getValue(FluxValues.NEXT_FORK, height)) {
+            return 1_000_000L;
+        }
+        return MAX_STEPS.get(atVersion(height));
+    }
+```
+
+## Compatibility
```

Comment:
What about the adjustmennt for the `createAT` to allow large SCs? Shouldn't it be mentioned here also?

Reply:
Sure, just did it.